src/components/Avatar/AvatarMenu.scss

```diff
@@ -80,6 +82,7 @@ $avatar-menu-size: 52px;
             .uir-avatar-menu-item a:focus,
             .uir-avatar-menu-item a:active {
                 display: inline-block;
+                flex: 1;
```

Comment:
I _think_ IE11 needs this to be `flex: 1 0 0%`

Reply:
https://caniuse.com/#feat=flexbox it's mentioned somewhere in here

Reply:
Good call - fixed!